File: app/src/main/java/com/cappielloantonio/tempo/github/Github.java
package com.cappielloantonio.tempo.github;

public class Github {
    private static final String OWNER = "eddyizm";
    private static final String REPO = "Tempus";

    public String getUrl() {
        return "https://api.github.com/";
    }

    public static String getOwner() {
        return OWNER;
    }

    public static String getRepo() {
        return REPO;
    }
}

File: app/src/main/java/com/cappielloantonio/tempo/github/Github.kt
package com.cappielloantonio.tempo.github

object Github {
    private const val OWNER = "eddyizm"
    private const val REPO = "Tempus"

    @JvmStatic
    fun getUrl(): String {
        return "https://api.github.com/"
    }

    @JvmStatic
    fun getOwner(): String {
        return OWNER
    }

    @JvmStatic
    fun getRepo(): String {
        return REPO
    }
}
